test/unit/dialog-controller.spec.js

```diff
@@ -16,32 +14,38 @@ describe('the Dialog Controller', function () {
     });
   }
 
-  beforeEach(function () {
-    new Container().makeGlobal();
-    settings = { name: 'Test' };
-    dialogController = new DialogController(renderer, settings, Function.prototype, Function.prototype);
+  function createDialogController(settings = {}) {
+    const dialogController = new DialogController(renderer, settings, Function.prototype, Function.prototype);
     dialogController.viewModel = {};
     dialogController.controller = {
       unbind() { }
     };
     dialogController.renderer.hideDialog = Function.prototype;
```

Comment:
I just realized this was in the test (and I know it's not from your code or this PR), not sure when it was added but something feels wrong about this.  We can clean it up later but seems like a weird thing to do :)